java/memory/src/main/java/org/apache/arrow/memory/AllocationManager.java

```diff
@@ -205,11 +205,11 @@ private void release(final BufferLedger ledger) {
     private final AtomicInteger bufRefCnt = new AtomicInteger(0); // start at zero so we can
     // manage request for retain
     // correctly
-    private final long lCreationTime = System.nanoTime();
```

Comment:
Just curious, the rule disallows single lower case character before upper case?

Reply:
this one was MemberNameCheck, but I relaxed it to allow allow for single lower case before uppercase since there were some other instances that seemed fine. I just forgot to revert these changes, I'll do that now.